apps/client/src/pages/home/Home.page.tsx

```diff
@@ -89,14 +128,35 @@ export function Home(): ReactElement {
           )}
         </div>
         {/* TODO: implement input validation. add max length from constants (some commits needed from previous branches) */}
+        {/* Player handle(username) input and join/create button */}
         <Input
           value={playerName}
           onChange={(e): void => setPlayerName(e.target.value)}
+          state={
+            !playerName
+              ? InputState.Neutral
+              : isValidPlayerName
+              ? InputState.Valid
+              : InputState.Invalid
```

Comment:
Can you seen if these nested turnery operations can be refactored so we can reuse the logic? so that we can simplify here and in the join room too.

Reply:
Done 4dd5220f69543cb0e398465f33091ddb484e36d7